extensions/quote/__init__.py

```diff
@@ -0,0 +1,118 @@
+"""Quotes from database"""
+import logger
+try:
+    import pymysql.cursors
+    PYMYSQL = True
+except ImportError:
+    import dep
+    if dep.ask("pymysql"):
+        try:
+            import pymysql.cursors
+            PYMYSQL = True
+            logger.info("pymysql Installed!", "green")
+        except ImportError:
+            logger.error("Failed to install pymysql")
+    else:
+        logger.error("Quoter will be unavailable as it requires pymysql")
+        PYMYSQL = False
+
+import asyncio
+import random
+import discord
+import tokens
+import argparse
+
+class BotExtension:
+    """Quoter"""
+    def __init__(self, bot):
+        self.name = "Quote System"
+        self.author = "nameless + Brett"
+        self.version = "1.0"
+        self.bot = bot
+        self.active = PYMYSQL
+        self.disable_during_test = False
+
+    def __register__(self):
+        return {
+            "quote" : {
+                "function": self.quote,
+                "roles" : ["@everyone"]
+            },
+            "save" : {
+                "function" : self.save,
+                "roles" : ["active"]
+            }
+        }
+
+    async def quote(self, args, message):
+        """Pulls Quotes from the Database"""
+        parser = argparse.ArgumentParser(description=self.quote.__doc__)
+        parser.add_argument("user", nargs="?", default="%", help="user to quote") #what is an % meant to be, a wildcard, it will match anything
+        args = await self.bot.parse_args(parser, args, message)
+        if args != False:
+            id = str(self.bot.get_from_tag(args.user))
```

Comment:
use something like uid instead of id, `id` is special in python and should not be overwritten